server/models/product.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+class Product {
+
+
+    constructor(id, farmerID, name, description, category) {
+
+        this.id = id;
+        this.farmerID = farmerID;
+        this.name = name;
+        this.description = description;
+        if (Object.values(Product.Categories).includes(category))
+            this.category = category;
+
+    }
+
+    static fromMongoJSON(json) {
+        return new Product(json._id, json.farmerID, json.name, json.description, json.category);
+    }
+
+    static Categories = {
+        Fruit: "fruit",
```

Comment:
Move the categories outside of the model, renaming it like: ProductCategory.
Use this standard for string enums:
`const Color = {
   RED: "value",
   BLUE: "value",
   GREEN: "value",
   YELLOW: "value"
};`